src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.WriteEntry.File.Tests.cs

```diff
@@ -206,5 +208,84 @@ public void Add_SymbolicLink(TarEntryFormat format, bool createTarget)
                 Assert.Null(reader.GetNextEntry());
             }
         }
+
+        [Theory]
+        [MemberData(nameof(WriteEntry_CopyArchive_Data))]
+        public void WriteEntry_CopyArchive(TestTarFormat testFormat, string testCaseName, CompressionMethod compressionMethod, bool copyData)
+        {
+            TarEntryFormat entryFormat = GetEntryFormatForTestTarFormat(testFormat);
+            string pathWithExpectedFiles = GetTestCaseUnarchivedFolderPath(testCaseName);
+            if (!Path.EndsInDirectorySeparator(pathWithExpectedFiles))
+            {
+                pathWithExpectedFiles = pathWithExpectedFiles + Path.DirectorySeparatorChar;
+            }
```

Comment:
This already exists in PathInternal:
https://github.com/dotnet/runtime/blob/78065413b2d1b4f0ed26343567379e992a3e26ee/src/libraries/Common/src/System/IO/PathInternal.cs#L18-L19

Reply:
Ah, thanks for the reminder.